{{cookiecutter.beat}}/README.md

```diff
@@ -0,0 +1,36 @@
+# {{cookiecutter.beat|capitalize}}
+
+Welcome to {{cookiecutter.beat}}.
+
+To get running with your beat, run the following commands:
+
+```
+glide init
+glide update
```

Comment:
"glide update --no-recursive" avoids to duplicate libbeat's dependencies. 
The libbeat's dependencies are already in the libbeat's vendor folder.